Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Get.ts

```diff
@@ -11,6 +11,11 @@ export async function getDocumentById({ id }: GetRequest, client: PoolClient): P
   try {
     const queryResult: QueryResult = await client.query(findDocumentByIdSql(id));
```

Comment:
I put a breakpoint on here and ran in debug mode. When it paused here, I renamed `documents` to `documents_2`. And it _did_ throw an appropriate error that I can see in the debugger. However, it just gave me a 500 response and did not log the error.

![image](https://user-images.githubusercontent.com/9324390/193860335-4a2a2ea2-f1be-48b2-8d51-58af7dfcc1c4.png)

Reply:
This is only true during the runtime of Meadowlark, once it's restarted, you will no longer see an error